PSSSRS/public-functions/Get-RSAdminSiteUrl.ps1

```diff
@@ -0,0 +1,63 @@
+<#
+Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+Licensed under the MIT License (MIT)
+#>
+
+function Get-RSAdminSiteUrl
+{
+<#
+.SYNOPSIS
+Get Admin Site Url

```

Comment:
probably a small comment that this method only returns a site when SSRS is SharePoint integrated